lib/rbs_rails/active_record.rb

```diff
@@ -166,6 +167,41 @@ def reload_#{a.name}: () -> #{type_optional}
         end.join("\n")
       end
 
+      private def generated_association_methods
+        # @type var sigs: Array[String]
+        sigs = []
+
+        # Needs to require "active_storage/engine"
+        if klass.respond_to?(:attachment_reflections)
```

Comment:
`attachment_reflections` will be defined after "active_storage/engine" is required, and will not be defined if it is not required.

https://github.com/rails/rails/blob/3be6c2d67c0dcbefa1c252943b0188b384403ec1/activestorage/lib/active_storage/reflection.rb#L43